core/src/verify.rs

```diff
@@ -1626,6 +1650,21 @@ mod test {
         assert!(csv.verify_reserved_state(&invalid_rs).is_ok());
     }
 
+<<<<<<< HEAD
```

Comment:
You have not resolved the conflicts here. Please make sure that you need to run `cargo fmt` and `cargo clippie and run all tests before submitting your PR. The documentation is on the following link.

[development guide](https://github.com/postech-dao/simperby/blob/main/docs/dev.md)

Reply:
I will make it a habit to run tests before submitting a PR!